packages/formik/src/Formik.tsx

```diff
@@ -904,7 +944,7 @@ export function useFormik<Values extends FormikValues = FormikValues>({
   );
 
   const getFieldProps = React.useCallback(
-    (nameOrOptions): FieldInputProps<any> => {
+    (nameOrOptions: any): FieldInputProps<any> => {
```

Comment:
Should this be `string | FieldHookConfig<any>` as used in `useField`?

Reply:
yes